docker-compose.yml

```
@@ -0,0 +1,18 @@
+version: '2.4.1'
+services:
+  api_service:
+    image: to_do_app:v1
+    restart: always
+    ports:
+      - "2030:2030"
```

Można tak spróbować - wtedy na Twoim komputerze będziesz to miała dostępne na `localhost:7000`
```suggestion
      - "7000:2030"
```